stories/Input.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { boolean, select, text, withKnobs } from '@storybook/addon-knobs'; // eslint-disable-line import/no-extraneous-dependencies
+import Input from '../src/components/Input';
+import { CONTEXTS, INPUT_TYPES, SIZES } from '../src/constants';
+
+storiesOf('Input', module)
```

Comment:
Would be nice to add a note like "Any other properties supplied will be spread to the underlined `<input />`  element"

Reply:
Can we maybe just add a code comment for now and add @storybook/addon-notes later in a new branch?

Reply:
sure, just maybe worth to add a task about it in order not to miss it